tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "sourceMap": true,
     "declaration": true,
     "declarationMap": false,
-    "types": ["node", "jest"],
+    "types": ["node", "vitest/globals"],
```

Comment:
@kibertoad is it intentional that we don't import the whole `"vitest"` as it was previously with `"jest"`?

Reply:
vitest/globals work the same way, it exposes the global namespaced methods that don't need to be imported, it's just a different naming convention